dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/BomTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.dashboard;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.List;
+
+import org.eclipse.aether.artifact.Artifact;
+import org.eclipse.aether.artifact.DefaultArtifact;
+import org.eclipse.aether.resolution.ArtifactDescriptorException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import com.google.cloud.tools.opensource.dependencies.RepositoryUtility;
+
+public class BomTest {
+
+  @Test
+  public void testArtifactsExist() throws ArtifactDescriptorException, IOException {
+    DefaultArtifact bom = new DefaultArtifact("com.google.cloud:cloud-oss-bom:pom:1.0.0-SNAPSHOT");
```

Comment:
Thanks. I'm going to see if we can get #485 in first, since that will allow this test to read from the current code instead of the maven repo.

Reply:
485 has been merged to master. The change removed a necessity of installing cloud-oss-bom locally.

@vgrankina Here is example code to load a BOM with relative path: https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/485/files#diff-c64c027455014b27d3fae482725a3938R134

Reply:
test is updated